ext/date/php_date.c

```diff
@@ -3386,7 +3408,7 @@ PHP_FUNCTION(timezone_open)
 	php_timezone_obj *tzobj;
 
 	ZEND_PARSE_PARAMETERS_START(1, 1)
-		Z_PARAM_STR(tz)
+		Z_PARAM_PATH_STR(tz) /* To prevent nul bytes */
```

Comment:
`nul` looks wrong, it's either `nil` or `null` or `\0`; and I prefer `nil` the least. The error message uses `null`.